src/interop/extendAtomicInteger/Rationale.md

```diff
@@ -0,0 +1,11 @@
+Correct answer: **d) Will not compile**
+
+* Compilation error is very strange: `Class 'MyAtomicInteger' is not abstract and does not implement abstract base class member public abstract fun toByte(): Byte defined in java.util.concurrent.atomic.AtomicInteger`
```

Comment:
is there a KT issue on YouTrack for this?

Reply:
I don't think so, at least I didn't created it.